commit 1d4d6da535be97b710e87a33c4828c97c36eee21
Author: Alexander Schmidt <alexs@linux.vnet.ibm.com>
Date:   Tue Jun 23 10:30:04 2009 -0700

    IB/ehca: Bump version number
    
    Increment version number for DMEM toleration.
    
    Signed-off-by: Alexander Schmidt <alexs@linux.vnet.ibm.com>
    Signed-off-by: Roland Dreier <rolandd@cisco.com>

Index: ofa_kernel-1.5/drivers/infiniband/hw/ehca/ehca_main.c
===================================================================
--- ofa_kernel-1.5.orig/drivers/infiniband/hw/ehca/ehca_main.c	2009-07-24 04:33:15.000000000 -0400
+++ ofa_kernel-1.5/drivers/infiniband/hw/ehca/ehca_main.c	2009-07-24 04:34:44.000000000 -0400
@@ -52,7 +52,7 @@
 #include "ehca_tools.h"
 #include "hcp_if.h"
 
-#define HCAD_VERSION "0026"
+#define HCAD_VERSION "0028"
 
 MODULE_LICENSE("Dual BSD/GPL");
 MODULE_AUTHOR("Christoph Raisch <raisch@de.ibm.com>");
